test/Test.hs

```diff
@@ -1,6 +1,23 @@
-import MPrelude
+import           Devtools.Files
+import           Devtools.Prelude
+import           Test.Tasty
+import           Test.Tasty.MGolden
 
+import qualified Data.List                     as List
+import qualified Data.Text                     as Text
 import qualified Devtools
+import qualified System.Path                   as Path
 
 main :: IO ()
-main = Devtools.main Devtools.defaultConfig
+main = do
+  files <- List.sort <$> findHaskellFiles [Path.relDir "src"]
```

Comment:
@epicallan Did you make it so the existing `src/**/*.hs` files became part of the golden test? If so, this is pretty cool.

Reply:
Yes, you specify which directories you want to format. Thanks.

Thinking about this some more I realise I could have used this glob package `http://hackage.haskell.org/package/Glob` for same functionality.